README.md

```diff
@@ -9,31 +9,155 @@ is [Disnake](https://github.com/DisnakeDev/disnake/).
 Did you ever want to experience the information overload and stress of a phone operator?
 Well, then you're at the right place!
 
-We've created a first stressful Discord game, in which you play a phone operator for a restaurant.
+We've created the first stressful Discord game, in which you play a phone operator for multiple restaurants.
 No matter what dishes, customers, or special orders you have to serve, don't lose focus, or you might get overwhelmed by
 the information overload!
 
-## Running the bot
+## Table of contents
+1. [Getting Started](https://github.com/SincereSingularities/SincereSingularities?tab=readme-ov-file#getting-started)
+2. [Video Presentation](https://github.com/SincereSingularities/SincereSingularities?tab=readme-ov-file#video-presentation)
+3. [Installation](https://github.com/SincereSingularities/SincereSingularities?tab=readme-ov-file#installation)
+4. [Running](https://github.com/SincereSingularities/SincereSingularities?tab=readme-ov-file#running)
+5. [Gameplay Tutorial](https://github.com/SincereSingularities/SincereSingularities?tab=readme-ov-file#gameplay)
+6. [Discord Bot Installation Guide](https://github.com/SincereSingularities/SincereSingularities?tab=readme-ov-file#discord-bot-installation-guide)
+7. [MongoDB Community Edition Installation Guide](https://github.com/SincereSingularities/SincereSingularities?tab=readme-ov-file#mongodb-community-edition-installation-guide)
+8. [Credits](https://github.com/SincereSingularities/SincereSingularities?tab=readme-ov-file#credits)
 
-1. Python 3.11 is [recommended](https://github.com/DisnakeDev/disnake/pull/1135#issuecomment-1847303628).
+
+
+## Getting Started
+
+1. Python 3.11 is [recommended](https://github.com/DisnakeDev/disnake/pull/1135#issuecomment-1847303628). Python 3.10 - 3.12 will probably work.
 2. Setup a [Discord Bot](https://docs.disnake.dev/en/stable/discord.html). <!-- TODO: explain better once we have an idea on how the bot works (e.g. what permissions are required) -->
-3. clone the repository:
+3. Clone the repository :
    ```shell
    git clone https://github.com/SincereSingularities/SincereSingularities/
    cd SincereSingularities
    pip install -e .
    ```
-4. Set the `BOT_TOKEN` environment variable to your Token using the `.env` file.
-5. Run The Game:
+4. Setup and run [MongoDB Community Edition](https://www.mongodb.com/docs/manual/administration/install-community/)
+5. Set the `BOT_TOKEN`, `DB_HOST`, `DB_PORT` and `DB_NAME` environment variables using the `.env` file.
+6. Run The Game:
    ```shell
    python -m sincere_singularities
    ```
-6. Play!
+7. Play!
```

Comment:
Why is this here if it is repeated below?

Reply:
This is a quick guide. Its to give the user an overview on whats needed to run the bot, and can be used to expand/translate the installation to the users preferred methods (e.g. a package manager which isnt `pip`).

A more complete installation/running guide can be seen below. This is a step-by-step guide, which you cant do anything wrong with.

Reply:
Ok. Maybe you could also link to the appropriate sections.

Reply:
Sure.